bin/test-local.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -o xtrace
+set -o errexit
+set -o nounset
+
+S3_BUCKET="${S3_BUCKET:-quickstart-heptio-com}"
```

Comment:
I'd like some docs around making this bucket, or making it as part of the script